2023-06-15/10:59:08		preprocessing: Error in initialising Preprocessor object - raw data cannot be read
2023-06-15/10:59:34		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:12:59		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:14:09		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:17:37		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:17:38		preprocessing: Error in preprocessing - list index out of range
2023-06-15/11:25:24		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:25:24		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-15/11:32:49		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/11:32:49		preprocessing: data successfully preprocessed and saved
2023-06-15/17:20:20		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-15/17:20:20		preprocessing: data successfully preprocessed and saved
2023-06-16/20:04:43		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-16/20:04:43		preprocessing: Error in preprocessing - [Errno 13] Permission denied: './data/preprocessed.csv'
2023-06-16/20:05:34		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-16/20:05:34		preprocessing: data successfully preprocessed and saved
2023-06-17/07:36:12		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/07:36:13		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:18:56		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:18:57		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:19:08		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:19:09		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:19:29		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:19:29		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:20:03		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:20:03		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: 'float' object has no attribute 'replace'
2023-06-17/21:20:17		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:20:17		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:21:10		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:21:11		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:22:44		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:22:44		preprocessing: Error in preprocessing - Inside Transformation object:  inside transform function: In Transformation.py: inside find_total_pixel: list index out of range
2023-06-17/21:24:58		preprocessing: raw data read successfully and Preprocessor object initialised
2023-06-17/21:24:58		preprocessing: data successfully preprocessed and saved
